openassessment/xblock/config_mixin.py

```diff
@@ -128,3 +152,17 @@ def is_mobile_support_enabled(self):
         # .. toggle_creation_date: 2020-10-14
         # .. toggle_tickets: https://github.com/edx/edx-ora2/pull/1445
         return settings.FEATURES.get('ENABLE_ORA_MOBILE_SUPPORT', False)
+
+    @cached_property
+    def is_rubric_reuse_enabled(self):
+        """
+        Return a boolean indicating the reuse of rubric feature is enabled or not.
+        """
+        # .. toggle_name: FEATURES['ENABLE_ORA_RUBRIC_REUSE']
+        # .. toggle_implementation: WaffleFlag
+        # .. toggle_default: False
+        # .. toggle_description: Set to True to enable the reuse of rubric feature
+        # .. toggle_use_cases: vip
```

Comment:
I think our use case is actually monitored rollout: https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html#use-case-3-ops-monitored-rollout